jetty-http/src/main/java/org/eclipse/jetty/http/pathmap/ServletPathSpec.java

```diff
@@ -270,6 +284,44 @@ private boolean isWildcardMatch(String path)
         return false;
     }
 
+    @Override
+    public MatchedPath matched(String path)
+    {
+        switch (_group)
+        {
+            case EXACT:
+                if (_declaration.equals(path))
+                    return MatchedPath.from(path, null);
```

Comment:
This should be a final field as it will always be the immutable `MatchedPath.from(_declaration, null)`

Reply:
final where?  if you have multiple PathSpecs, you have multiple final fields, which means a lookup/map again.